package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-calculation-service",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "3.1.3",
+  "version": "3.2.0",
```

Comment:
Can we move this to `3.4.0`. In the last merge it went from `3.3.0` to `3.1.3` instead of `3.3.1`